docs/gs_installation_choices.md

```diff
@@ -0,0 +1,18 @@
+# Welcome To OpenVoice OS Getting Started User Guide
+# Installation Choices
+
+You can find a pdf version of a getting started guide [here](https://github.com/OpenVoiceOS/ovos_assets/raw/master/printables/device-getting-started-guide.pdf)
+
+## Where to get OVOS?
+OVOS is in early stages, we publish our Raspberry Pi images for download but expect new bugs and new fixes on every release, we are not yet stable!
+
+These images are development images in alpha stage, bugs and incomplete features are guaranteed.
+
+### Install choices
+You can install OVOS either as an image, container, or manually.
+
+There are currently three image choices for OVOS.  Buildroot, Manjaro and Picroft.  You can also build images from scratch using Buildroot and Manjaro
```

Comment:
mentioned this in chats, but we can not use 'picroft' due to copyright issues

Reply:
yeah, I had gone through previously and removed them, but in my process of trying to get up to date with the community-docs master, I seem to have reintroduced some.